### DISCUSSION TITLE
Solid CG Chairs Rotation Schedule

### DISCUSSION THREAD

**Discussion by VirginiaBalseiro** [solid/specification#618] [Announcements]
The co-chairs of the CG follow a rotation schedule. The following table shows the schedule for upcoming meetings.

| Date  | Chair  |  Agenda |
|---|---|---|
|2024-01-31   | Virginia Balseiro   | |
|2024-02-07   | Hadrian Zbarcea   | |
|2024-02-14   | Hadrian Zbarcea  | |
|2024-02-21 | Virginia Balseiro   | |
|2024-02-28   | Michiel de Jong   | |
|2024-03-06   | HadrianZbarcea  | |
|2024-03-13   | Virginia Balseiro  | |
|2024-03-20   | Michiel de Jong | |
|2024-03-27   | Hadrian Zbarcea | |
|2024-04-03   | Virginia Balseiro | |
|2024-04-10   | Hadrian Zbarcea | |
|2024-04-17   | Virginia Balseiro  | |
|2024-04-24   | Virginia Balseiro  | |
|2024-05-01   | Hadrian Zbarcea | |
|2024-05-08   | Virginia Balseiro  | |
|2024-05-15   | Michiel de Jong |  https://hackmd.io/oZjh0CLOS3uolsnT8hUSFg |
|2024-05-22   | Hadrian Zbarcea | |
|2024-05-29   | Michiel de Jong | https://hackmd.io/f77eAD-VQ6eV7eJGkWPJog |
|2024-06-05   | Virginia Balseiro | |
|2024-06-12   | Hadrian Zbarcea  | |
|2024-06-19   | Michiel de Jong | https://hackmd.io/XJziaa1KT5GcH9ZAHzRgKA |
|2024-06-26   | Michiel de Jong |https://hackmd.io/wr4j84iMSgqdnp1Eiekg8w |
|2024-07-03   | Virginia Balseiro | |
|2024-07-10   | Hadrian Zbarcea  | |
|2024-07-17   | Hadrian Zbarcea  | |
|2024-07-24   |  | |
|2024-07-31   | Hadrian Zbarcea  | open |
|2024-08-07   |  Virginia Balseiro | |
|2024-08-14   | Hadrian Zbarcea | open |
|2024-08-21   | Virginia Balseiro | |
|2024-08-28   | Michiel de Jong | https://hackmd.io/6bidAPgeQaOm3XH7HfZ_IQ |
|2024-09-04   | | |
|2024-09-11   | Michiel de Jong | https://hackmd.io/OO09JtrDTLuDi4v4fAd2Sg |
|2024-09-18   | Virginia Balseiro | |
|2024-09-25   | | |
|2024-09-27   | Virginia Balseiro | TPAC: https://hackmd.io/1QInJs0-QS62qHPQbUTV-Q |
|2024-10-02   | Michiel de Jong  | https://hackmd.io/z4M0FTWXSFGRfh0r0bRPAw |
|2024-10-09   | Hadrian Zbarcea  | https://hackmd.io/@solid/SJc9B-fJJl |
|2024-10-16   | Virginia Balseiro  | |
|2024-10-23   | Michiel de Jong  | |
|2024-10-30   | Hadrian Zbarcea  | |
|2024-11-06   | Virginia Balseiro  | |
|2024-11-13   | Michiel de Jong  | |
|2024-11-20   | Hadrian Zbarcea  | |
|2024-11-27   | Michiel de Jong  | https://hackmd.io/bzhmaEkkRSWknliDsoaWcw |
|2024-12-04   | Hadrian Zbarcea  | |
|2024-12-11   | Michiel de Jong  | https://hackmd.io/0-R4mnG1QJeMkUmxvN_qGg |
|2024-12-18  | Hadrian Zbarcea  | |
|2024-12-25   | (canceled due to Christmas)  | |
|2025-01-01   | (canceled due to New Year's Day)  | |

**Comment by michielbdejong** [comment]
Due to limited availability of the chairs and possibly other people, should we skip 17, 24 and 31 July as a summer break?
If people want they can do a more informal video chat without official agenda / chairing / minutes.

**Reply by hzbarcea** [reply]
I can chair 07/17. I am ok with a break too, although I believe August would be the more problematic month, especially in EU.

**Reply by VirginiaBalseiro** [reply]
The group can continue with the meetings even with the chairs absent, people can self-organize and take minutes, we have done this many times in the past. In any case, I'll be around in August.

**Comment by michielbdejong** [comment]
Continued in https://github.com/solid/specification/discussions/703
